coursework2/src/main/java/oursework2/service/ExaminerServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package oursework2.service;
+
+import org.springframework.stereotype.Service;
+import oursework2.exception.RequestLimitExceededException;
+import oursework2.exception.RequiredNumberOfQuestionsIsMissing;
+import oursework2.model.Question;
+
+import java.util.*;
+
+@Service
+public class ExaminerServiceImpl implements ExaminerService {
+    private final JavaQuestionService javaQuestionService;
```

Comment:
```suggestion
    private final QuestionService javaQuestionService;
```
о интерфейсу надо делать инъекцию, а не реализации (см. полиморфизм, L в SOLID)